projects/batfish-common-protocol/src/main/java/org/batfish/datamodel/routing_policy/expr/MatchAsPath.java

```diff
@@ -63,4 +55,13 @@ public int hashCode() {
   public void setExpr(AsPathSetExpr expr) {
     _expr = expr;
   }
+
+  @Override
+  public String toString() {
+    return MoreObjects.toStringHelper(MatchAsPath.class)
+        .omitNullValues()
+        .add("comment", getComment())
+        .add("expr", _expr)
```

Comment:
Can you use the `PROP_XXX` constants here? If they don't exist in this class, I'd suggest making them and referencing them here (and in the getters and setters).